### ISSUE TITLE
Static Site Generation with React and Webpack

### ISSUE THREAD

**Issue by geyang** [jxnblk/blog#28]
http://jxnblk.com/writing/posts/static-site-generation-with-react-and-webpack/

The site looks super clean, but the aesthetics shouldn't override the usability. The code blocks lack highlighting, and make it really hard to parse. 

Also although the tutorial is very well-writhing, having a git repository would be very helpful. A lot of us want to just see the finished code in the end.

Thanks for the post!

Ge

**Comment by jxnblk** [comment]
Thanks, there is a repo linked to in the article.
